docs/src/test/scala/docs/http/scaladsl/HttpAppExampleSpec.scala

```diff
@@ -4,10 +4,12 @@
 
 package docs.http.scaladsl
 
+import com.github.ghik.silencer.silent
 import docs.CompileOnlySpec
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 
+@silent("deprecated")
```

Comment:
Is there a particular reason to keep the docs examples around?

Reply:
Dropped them now.